### PR TITLE
Added subprocess in translateSimple

### PR DESCRIPTION
**Pull request by jatinluthra14**

### CODE REVIEW

**Comment by sushain97** [comment]
Why is the build failing?

**Comment by jatinluthra14** [comment]
See the logs. I think it's not able to find a command

**Comment by unhammer** [comment]
As far as I can tell, this changes AnalyzeHandler and GenerateHandler; it looks like the issue was about document translation.

**Comment by jatinluthra14** [comment]
It was for translateSimple() as written in issue

**Comment by unhammer** [comment]
translateSimple() doesn't use analyse/generatehandler. It's a helper function.

the issue is called " Document translation should spawn processes".

File: translation_py32.py [hunk 1]
```diff
@@ -329,15 +329,20 @@ def translateSimple(toTranslate, commands):
...
 def translateDoc(fileToTranslate, fmt, modeFile, unknownMarks=False):
     modesdir = os.path.dirname(os.path.dirname(modeFile))
     mode = os.path.splitext(os.path.basename(modeFile))[0]
     if unknownMarks:
         cmd = ['apertium', '-f', fmt,       '-d', modesdir, mode]
     else:
         cmd = ['apertium', '-f', fmt, '-u', '-d', modesdir, mode]
-    proc = Popen(cmd, stdin=fileToTranslate, stdout=PIPE)
+
+    proc_in = yield gen.Task(proc.stdin.write, fileToTranslate)
```
Comment (unhammer) [inline translation_py32.py]: TypeError: write() only accepts bytes, unicode, and dict objects

Please test that your code works before you submit for review. If you need help testing, ask on IRC.

**Comment by unhammer** [comment]
doesn't work (undefined variable proc)

**Pull request closed without merging**